### PR TITLE
Add/update license headers

### PR DESCRIPTION
**Pull request by Tilka**
Seriously tired of copyright.

### CODE REVIEW

File: Source/Core/AudioCommon/aldlist.cpp [hunk 1]
```diff
@@ -1,3 +1,7 @@
+// Copyright 2009-2014 Dolphin Emulator Project
+// Licensed under GPLv2
```
Comment (Sonicadvance1) [inline Source/Core/AudioCommon/aldlist.cpp]: Recopywriting Creative Labs code?

File: Source/Core/Common/BitSet.h [hunk 2]
```diff
@@ -1,3 +1,7 @@
+// Copyright 2014-2015 Dolphin Emulator Project
```
Comment (Sonicadvance1) [inline Source/Core/Common/BitSet.h]: This file is public domain, not GPLv2

File: Source/Core/Common/ExtendedTrace.cpp [hunk 3]
```diff
@@ -1,3 +1,7 @@
+// Copyright 2008-2014 Dolphin Emulator Project
```
Comment (Sonicadvance1) [inline Source/Core/Common/ExtendedTrace.cpp]: This isn't our code.

File: Source/Core/Common/ExtendedTrace.h [hunk 4]
```diff
@@ -1,3 +1,7 @@
+// Copyright 2008-2014 Dolphin Emulator Project
```
Comment (Sonicadvance1) [inline Source/Core/Common/ExtendedTrace.h]: Again not ours.

File: Source/Core/Common/GekkoDisassembler.cpp [hunk 5]
```diff
@@ -1,3 +1,7 @@
+// Copyright 2014-2015 Dolphin Emulator Project
+// Licensed under GPLv2
```
Comment (Sonicadvance1) [inline Source/Core/Common/GekkoDisassembler.cpp]: Not ours

File: Source/Core/Common/GekkoDisassembler.h [hunk 6]
```diff
@@ -1,3 +1,7 @@
+// Copyright 2014 Dolphin Emulator Project
+// Licensed under GPLv2
```
Comment (Sonicadvance1) [inline Source/Core/Common/GekkoDisassembler.h]: Not ours

File: Source/Core/Common/SDCardUtil.cpp [hunk 7]
```diff
@@ -1,3 +1,7 @@
+// Copyright 2009-2014 Dolphin Emulator Project
+// Licensed under GPLv2
```
Comment (Sonicadvance1) [inline Source/Core/Common/SDCardUtil.cpp]: Definitely not ours

File: Source/Core/Core/DSP/DSPAssembler.cpp [hunk 8]
```diff
@@ -1,3 +1,7 @@
+// Copyright 2009-2014 Dolphin Emulator Project
```
Comment (Sonicadvance1) [inline Source/Core/Core/DSP/DSPAssembler.cpp]: Not ours

File: Source/Core/Core/DSP/DSPAssembler.h [hunk 9]
```diff
@@ -1,3 +1,7 @@
+// Copyright 2009-2014 Dolphin Emulator Project
```
Comment (Sonicadvance1) [inline Source/Core/Core/DSP/DSPAssembler.h]: Not ours

File: Source/Core/Core/DSP/DSPCore.cpp [hunk 10]
```diff
@@ -1,3 +1,7 @@
+// Copyright 2008-2014 Dolphin Emulator Project
```
Comment (Sonicadvance1) [inline Source/Core/Core/DSP/DSPCore.cpp]: not ours

File: Source/Core/Core/DSP/DSPCore.h [hunk 11]
```diff
@@ -1,3 +1,7 @@
+// Copyright 2008-2014 Dolphin Emulator Project
```
Comment (Sonicadvance1) [inline Source/Core/Core/DSP/DSPCore.h]: not ours

File: Source/Core/Core/DSP/DSPDisassembler.cpp [hunk 12]
```diff
@@ -1,3 +1,7 @@
+// Copyright 2008-2014 Dolphin Emulator Project
```
Comment (Sonicadvance1) [inline Source/Core/Core/DSP/DSPDisassembler.cpp]: not ours

File: Source/Core/Core/DSP/DSPDisassembler.h [hunk 13]
```diff
@@ -1,3 +1,7 @@
+// Copyright 2008-2014 Dolphin Emulator Project
```
Comment (Sonicadvance1) [inline Source/Core/Core/DSP/DSPDisassembler.h]: not ours

File: Source/Core/Core/DSP/DSPHWInterface.cpp [hunk 14]
```diff
@@ -1,3 +1,7 @@
+// Copyright 2008-2014 Dolphin Emulator Project
```
Comment (Sonicadvance1) [inline Source/Core/Core/DSP/DSPHWInterface.cpp]: not ours

File: Source/Core/Core/DSP/DSPHWInterface.h [hunk 15]
```diff
@@ -1,3 +1,7 @@
+// Copyright 2008-2014 Dolphin Emulator Project
```
Comment (Sonicadvance1) [inline Source/Core/Core/DSP/DSPHWInterface.h]: not ours

File: Source/Core/Core/DSP/DSPIntExtOps.h [hunk 16]
```diff
@@ -1,3 +1,7 @@
+// Copyright 2008-2014 Dolphin Emulator Project
```
Comment (Sonicadvance1) [inline Source/Core/Core/DSP/DSPIntExtOps.h]: not ours

File: Source/Core/Core/DSP/DSPIntUtil.h [hunk 17]
```diff
@@ -1,3 +1,7 @@
+// Copyright 2008-2014 Dolphin Emulator Project
```
Comment (Sonicadvance1) [inline Source/Core/Core/DSP/DSPIntUtil.h]: not ours

File: Source/Core/Core/DSP/DSPInterpreter.cpp [hunk 18]
```diff
@@ -1,3 +1,7 @@
+// Copyright 2008-2015 Dolphin Emulator Project
```
Comment (Sonicadvance1) [inline Source/Core/Core/DSP/DSPInterpreter.cpp]: not ours

File: Source/Core/Core/DSP/DSPMemoryMap.cpp [hunk 19]
```diff
@@ -1,3 +1,7 @@
+// Copyright 2008-2014 Dolphin Emulator Project
```
Comment (Sonicadvance1) [inline Source/Core/Core/DSP/DSPMemoryMap.cpp]: not ours

File: Source/Core/Core/DSP/DSPMemoryMap.h [hunk 20]
```diff
@@ -1,3 +1,7 @@
+// Copyright 2008-2014 Dolphin Emulator Project
```
Comment (Sonicadvance1) [inline Source/Core/Core/DSP/DSPMemoryMap.h]: not ours

File: Source/Core/Core/DSP/DSPStacks.cpp [hunk 21]
```diff
@@ -1,3 +1,7 @@
+// Copyright 2008-2014 Dolphin Emulator Project
```
Comment (Sonicadvance1) [inline Source/Core/Core/DSP/DSPStacks.cpp]: not ours

File: Source/Core/Core/DSP/DSPStacks.h [hunk 22]
```diff
@@ -1,3 +1,7 @@
+// Copyright 2008-2014 Dolphin Emulator Project
```
Comment (Sonicadvance1) [inline Source/Core/Core/DSP/DSPStacks.h]: not ours

File: Source/Core/Core/HW/BBA-TAP/TAP_Win32.h [hunk 23]
```diff
@@ -1,3 +1,7 @@
+// Copyright 2009-2014 Dolphin Emulator Project
```
Comment (Sonicadvance1) [inline Source/Core/Core/HW/BBA-TAP/TAP_Win32.h]: not ours.
Probably will have an issue when we convert to GPLv2+?

**Comment by Sonicadvance1** [comment]
I'll comment on more once you fix those ones. I don't want to pull your repo right now.

**Comment by Pringo** [comment]
I've noticed there's a discrepancy between the license headers in the source code and the copyright notice in the Dolphin about page. For this you have "Copyright 20XX-20XX Dolphin Emulator Project" while in Dolphin itself there is "Copyright (c) 2003-2014+ Dolphin Team". Should they both be attributed to Dolphin Team or is there a reason they are different?

File: Source/Core/AudioCommon/aldlist.cpp (hunk 1 above)
Comment (Tilka) [inline Source/Core/AudioCommon/aldlist.cpp]: There have been non-trivial bug fixes, afaict.

File: Source/Core/Core/HW/BBA-TAP/TAP_Win32.h (hunk 23 above)
Comment (Tilka) [inline Source/Core/Core/HW/BBA-TAP/TAP_Win32.h]: It's a header required for interoperability.

File: Source/Core/Common/GekkoDisassembler.cpp (hunk 5 above)
Comment (Tilka) [inline Source/Core/Common/GekkoDisassembler.cpp]: There have been numerous changes.

File: Source/Core/Common/GekkoDisassembler.h (hunk 6 above)
Comment (delroth) [inline Source/Core/Common/GekkoDisassembler.h]: @lioncash has rewritten a lot of it.

**Comment by delroth** [comment]
TBF I would just put "Copyright © 2003" everywhere. Having the current year is just an annoyance.

**Comment by Tilka** [comment]
2003 or the first year a file appeared?

**Comment by delroth** [comment]
Not sure. I don't really care as long as we don't have to continuously update these notices that have basically 0 legal value.

**Comment by BhaaLseN** [comment]
Didn't we have a similar conversation years back and decided on "the year we add a new file should have the current year" and "just leave the others as they are, they should be somewhat close to when the file was created"?
I'd prefer not having to update them every year.

**Comment by shuffle2** [comment]
I am scared to open the changes view in github's javascript viewer. Can someone summarize the commit in words? :P

**Comment by slx7R4GDZM** [comment]
It looks like it just changes the copyright year on the files to when they were first created and adds copyright text to files that were missing it.

**Comment by shuffle2** [comment]
@dolphin-emu-bot rebuild

**Comment by Tilka** [comment]
Included into #1826.

**Pull request closed without merging**